README.md

```diff
@@ -190,7 +190,7 @@ yotiClient.getActivityDetails(token).then((activityDetails) => {
 
 ```
 
-Where `yourUserSearchFunction` is a piece of logic in your app that is supposed to find a user, given a userId.
+Where `yourUserSearchFunction` is a piece of logic in your app that is supposed to find a user, given a _Remember Me Id_.
```

Comment:
If we're not doing camelcase, we should make it "Remember Me I**D**" 👍

Reply:
Corrected in #42 and rebased